Tests/TorusUtilsTests/SapphireTest.swift

```
@@ -336,4 +336,32 @@ final class SapphireTest: XCTestCase {
         try await testAggregrateLoginWithEmail(email: email)
     }
     */
+    
+    
+    func testGating() async throws {
+        let torus = TorusUtils(enableOneKey: true, network: .sapphire(.SAPPHIRE_MAINNET), clientId: "YOUR_CLIENT_ID")
+        let token = try generateIdToken(email: TORUS_TEST_EMAIL)
+
+        let verifierParams = VerifierParams(verifier_id: TORUS_TEST_EMAIL)
+
+        let nodeDetails = try await get_fnd_and_tu_data(verifer: "w3a-auth0-demo", veriferID: TORUS_TEST_EMAIL)
+
+        do {
+            let data = try await torus.retrieveShares(
+                endpoints: nodeDetails.getTorusNodeEndpoints(),
+                torusNodePubs: nodeDetails.getTorusNodePub(),
+                indexes: nodeDetails.getTorusIndexes(),
+                verifier: "w3a-auth0-demo",
+                verifierParams: verifierParams,
+                idToken: token
+            )
+            XCTAssert(false, "Should not pass")
```

Shouldn't this be success instead of failure?

---

It should fail because of invalid client id